Pages/Hotels/HotelHomePage.js

```diff
@@ -199,19 +201,19 @@ async clickListYourProperty() {
     await browser.pause(2000);
 }
 
-async switchingToPropertyHandle() {
-    const connectedHandle =await browser.getWindowHandle();
-
+async switchingToPropertyHandle(url) {
+    const connectedHandle = await browser.getWindowHandle();
+  
     const allHandles = await browser.getWindowHandles();
     for (const handle of allHandles) {
-        await browser.switchToWindow(handle);
-        const currentUrl = await browser.getUrl();
-        console.log(`current url -> ${currentUrl}\n\n`);
-        if (currentUrl.localeCompare('https://apps.expediapartnercentral.com/en_US/list?utm_medium=referral&utm_source=HCOM_US-en_US&utm_campaign=HomePage&utm_contentewd=pwa-header-btn&siteId=300000001&tpid=3001&eapid=1&langId=1033') === 0) {
-            return handle;
-        }
+      await browser.switchToWindow(handle);
+      const currentUrl = await browser.getUrl();
+      console.log(`current url -> ${currentUrl}\n\n`);
+      if (currentUrl.localeCompare(url) === 0) {
```

Comment:
To compare equality(===) is hectic sometimes, you can try compare contains(). It will be easier.